ecs-init/config/default_system_paths.go

```diff
@@ -1,4 +1,5 @@
 // +build !suse
+// +build !ubuntu
```

Comment:
I think it's more idiomatic to put these on one line (`!suse,!ubuntu`), but I'm not certain. I'm fine with leaving it as is if you prefer this.